exporter/opensearchexporter/metadata.yaml

```diff
@@ -2,5 +2,7 @@ type: opensearch
 
 status:
   class: exporter
+  stability:
+    development: [traces]
   codeowners:
```

Comment:
Should we add a line for the contrib distribution here to match the original README?

Reply:
The component is not in the contrib distro, see https://github.com/open-telemetry/opentelemetry-collector-releases/pull/391